lib/plan_executor/orch_client.rb

```diff
@@ -55,7 +69,19 @@ def send_request(targets, task, arguments, options = {})
       body[:description] = description if description
       body[:plan_job] = @plan_job if @plan_job
 
-      @client.run_task(body)
+      response = @http.post('internal/plan_task', body)
+      if response.status != 202
+        raise Bolt::Error.new(response.body['msg'], response.body['kind'], response.body['details'])
+      end
+      url = response.body.dig('job', 'id')
```

Comment:
And I guess adding a `post` function would make the usage here something like `url = post('internal/plan_task', body).dig('job', 'id')`